src/controllers/review.js

```diff
@@ -0,0 +1,81 @@
+const dishModel = require('../models/dish');
+
+const CustomError = require('../utils/customError');
+
+const addReviewToDish = async (req, res, next) => {
+  const { dishId } = req.params;
+  const { rating, comment } = req.body;
+  const reviewData = { rating, comment };
```

Comment:
where is customerId, you should store customer id who created the review
please check review schema

Reply:
I will fix it